go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -208,6 +210,12 @@ func runOvnKube(ctx *cli.Context) error {
 		ovncluster.StartMetricsServer(config.Kubernetes.MetricsBindAddress)
 	}
 
+	// Set up a watch on our config file; if it changes, we exit -
+	// (we don't have the ability to dynamically reload config changes).
+	if err := watchForChanges(configFile); err != nil {
+		logrus.Fatalf("unable to setup configuration watch: %v", err)
```

Comment:
@pecameron Sorry @alexanderConstantinescu made changes few days back to clean things up because of which you will need to now do `return fmt.Errorf("unable to setup configuration watch: %v", err)` to be consistent.

Reply:
@girishmg OK, made the change.